docs(webpack5): add new cache options (#3413)
webpack/webpack.js.org: src/content/configuration/other-options.md

W> Don't share the cache between calls with different options.

### `cache.idleTimeout`

`number = 10000`

> Time in `ms`. `cache.idleTimeout` denotes the time period after which the cache storing should happen. Defaults to `10000` which is equivalent to `10s`

Reviewer: No need to elaborate when it already says milliseconds
```suggestion
Time in milliseconds. `cache.idleTimeout` denotes the time period after which the cache storing should happen.
```

__webpack.config.js__

```javascript
module.exports = {
  //..
  cache: {
    idleTimeout: 10000
  }
};
```

> W> `cache.idleTimeout` is only available for when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`

Reviewer: Lets use simpler form
```suggestion
W> `cache.idleTimeout` is only available when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
```

### `cache.idleTimeoutForInitialStore`

`number = 0`

> Time in `ms`. `cache.idleTimeoutForInitialStore` is the time period after which the initial cache storing should happen. It defaults to `0` which is equivalent to `0s`.

Reviewer: ```suggestion
Time in milliseconds. `cache.idleTimeoutForInitialStore` is the time period after which the initial cache storing should happen.
```

__webpack.config.js__

```javascript
module.exports = {
  //..
  cache: {
    idleTimoutForInitialStore: 0
  }
};
```

> W> `cache.idleTimeoutForInitialStore` is only available for when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`

Reviewer: ```suggestion
W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#cachestore) is set to either `'pack'` or `'idle'`
```
